Makefile

```diff
@@ -50,7 +50,7 @@ upgrade: ## update the requirements/*.txt files with the latest packages satisfy
 	cat requirements/quality-extra.txt >> requirements/quality.txt
 	cat requirements/test-extra.txt >> requirements/test.txt
 	# Let tox control the Django version for tests
-	sed '/Django==/d' requirements/test.txt > requirements/test.tmp
+	sed '/^[Dd]jango==/d' requirements/test.txt > requirements/test.tmp
```

Comment:
This is a mostly unrelated boy-scout-rule cleanup of an issue that bit me in testing, not a relevant change for the migrations